src/main/cli/cli.c

```diff
@@ -1409,14 +1410,34 @@ static int cliParseSerialMode(const char *tok)
     return mode;
 }
 
+static int cliParseSerialOptions(const char *tok)
+{
+    int options = SERIAL_NOT_INVERTED;
```

Comment:
```
struct {
   const char name[4];
   int options;
} mode_map[] const = {{"8N1", SERIAL_STOPBITS_1 | SERIAL_PARITY_NO}, ... };
for(unsigned i = 0; i < ARRAYLEN(mode_map); i++) {
  if(!strncasecmp(tok, mode_map[i].name, sizeof(mode_map[i].name))
     return  mode_map[i].options;
}
// TODO - handle unknown options .,.. 
}

```

Reply:
Nice one @ledvinap!

Reply:
@wvarty: Feel like making this improvement?

Reply:
Can do. Should give me a chance to fix that typo in the docs too.

Reply:
@wvarty: Are you going to get around to changing this?

Reply:
@mikeller yep, sorry for the delay following up on this, the need for this change has become somewhat lower priority in terms of the use case.

I started making the change suggested above, and noticed a defect in the way I've written this. The original code checked if any serial parameters had changed, and if not, the original port settings were retained, and passthrough was executed. With my changes, I re-open the port regardless of any changes to serial params, and I used a default of `SERIAL_NOT_INVERTED` which means that ports that were originally opened as inverted would be changed to non-inverted as pert of entering passthrough.

I'm 90% there with local changes to both, so I'll try and knock it over this week sometime :)

Reply:
Good spot! I didn't notice this change of approach in the review, but I think it is a better approach if the port is left as is if no extra port parameters are specified - this is what will be needed in a majority of use cases.